glotpress.php

```diff
@@ -35,6 +35,35 @@
 define( 'GP_PLUGIN_FILE', __FILE__ );
 define( 'GP_PATH', dirname( __FILE__ ) . '/' );
 define( 'GP_INC', 'gp-includes/' );
+define( 'GP_WP_REQUIRED_VERSION', '4.4' );
+
+// This adds a row after GlotPress in the plugin page IF an incompatible version of WordPress is running.
+function gp_admin_notices() {
+	global $wp_version;
+	
+	$screen = get_current_screen();
+	
+	if( 'plugins' == $screen->id ) {
+?>
+    <div class="error">
+		<p><?php _e( 'GlotPress Disabled &#151; You are running an unsupported version of WordPress.', 'glotpress' ); ?></p>
+
+		<p><?php echo sprintf( __( 'GlotPress requires WordPress %s or later and has detected you are running %s. Upgrade your WordPress install or deactivate the GlotPress plugin to remove this message.', 'glotpress' ), GP_WP_REQUIRED_VERSION, $wp_version ); ?></p>
```

Comment:
* Both sentences should be in one paragraph, if we want a line break let's use `<br>`.
* The placeholders need a translator comment, see https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables. In this case numbered placeholders (`%1$s`) should be used.
* Both variables should be `esc_html`'ed.